ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-builder.view.js

```diff
@@ -59,29 +59,47 @@ module.exports = Marionette.LayoutView.extend({
   },
   onBeforeShow() {
     this.$el.toggleClass('is-sortable', this.options.isSortable || false)
-    this.filterOperator.show(
-      DropdownView.createSimpleDropdown({
-        list: [
-          {
-            label: 'AND',
-            value: 'AND',
-          },
-          {
-            label: 'OR',
-            value: 'OR',
-          },
-          {
-            label: 'NOT AND',
-            value: 'NOT AND',
-          },
-          {
-            label: 'NOT OR',
-            value: 'NOT OR',
-          },
-        ],
-        defaultSelection: [this.model.get('operator') || 'AND'],
-      })
-    )
+    if (this.options.isFormBuilder === true) {
+      this.filterOperator.show(
+        DropdownView.createSimpleDropdown({
+          list: [
+            {
+              label: 'AND',
+              value: 'AND',
+            },
+            {
+              label: 'OR',
+              value: 'OR',
+            },
+          ],
+          defaultSelection: [this.model.get('operator') || 'AND'],
+        })
+      )
+    } else {
+      this.filterOperator.show(
+        DropdownView.createSimpleDropdown({
```

Comment:
✏️ There's a bit of duplication here. I recommend encapsulating the common bits. One approach:
```
const extraLabels = [
              {
              label: 'NOT AND',
              value: 'NOT AND',
            },
            {
              label: 'NOT OR',
              value: 'NOT OR',
            },
]

const showExtraLabels = !this.options || this.options.isFormBuilder !== true

const labels = {
          list: [
            {
              label: 'AND',
              value: 'AND',
            },
            {
              label: 'OR',
              value: 'OR',
            },
            ...(showExtraLabels ? extraLabels : [])
        ]
}
```